module/data/item/equipment.mjs

```diff
@@ -19,12 +19,14 @@ export default class EquipmentData extends PhysicalItemTemplate.mixin(
                 label: "ED.Item.Equipment.consumable"
             } ),
             // different ammo types are availabel see issue #
-            ammoType: new fields.StringField( {
+            ammunition: new fields.SchemaField( {
```

Comment:
I can still find 3 occurrences of "ammoType" on this branch... They should be corrected

Reply:
fixed it